tutorials/modernize-rfc-receiver/modernize-rfc-receiver.md

```diff
@@ -0,0 +1,168 @@
+---
+author_name: Gabriel Alves
+author_profile: https://github.com/ggalvesdevsap
+parser: v2
+auto_validation: true
+time: 30
+tags: [  software-product>sap-integration-suite, software-product>cloud-integration, software-product>sap-process-integration, software-product>sap-process-orchestration, tutorial>license, tutorial>beginner  ]
+primary_tag: software-product>sap-integration-suite
+---
+
+# Learn how to modernize RFC Receiver Communications into API-Based protocols in Cloud Integration.
```

Comment:
Please rewrite tutorial title as:
Learn How to Modernize RFC Receiver Communications into API-Based Protocols in Cloud Integration